x-pack/plugin/security/cli/build.gradle

```diff
@@ -22,16 +22,18 @@ dependencyLicenses {
     mapping from: /bc.*/, to: 'bouncycastle'
 }
 
-if (project.inFipsJvm) {
-    test.enabled = false
-    testingConventions.enabled = false
-    // Forbiden APIs non-portable checks fail because bouncy castle classes being used from the FIPS JDK since those are
-    // not part of the Java specification - all of this is as designed, so we have to relax this check for FIPS.
-    tasks.withType(CheckForbiddenApis) {
-        bundledSignatures -= "jdk-non-portable"
-    }
-    // FIPS JVM includes many classes from bouncycastle which count as jar hell for the third party audit,
-    // rather than provide a long list of exclusions, disable the check on FIPS.
-    thirdPartyAudit.enabled = false
+rootProject.globalInfo.ready {
```

Comment:
This makes me wonder if it's worth running forbidden apis and third party audit at all on fips as there's no additional value.  As in centrally disable all of them so we don't ave all this fisp specific logic in build scripts. //cc @jkakavas 
The tests and testing conventions could be an `onlyIf`

Reply:
👍 I'm generally in favor of this. These checks are compile-time and really shouldn't be dependent on the runtime JVM. I would be in favor of disabling these across the board for FIPS JVMs.